VotingApplication/VotingApplication.Web.Api/Scripts/ResultChart.js

```diff
@@ -0,0 +1,92 @@
+﻿define('ResultChart', ['jquery', 'knockout', 'insight'], function ($, ko, insight) {
+    var chartCounter = 0;
+    function ResultChart($element) {
+        chartCounter++;
+
+        var chart;
+
+        this.drawChart = function (data, options) {
+            if (!data.length) return;
+
+            var dataUnchanged = false;
+            var barCount = 0;
+            for (var n = 0; n < data.length; n++) {
+                barCount += data[n].Data.length;
+                dataUnchanged = dataUnchanged &&
+                    chart && chart.series().length > n &&
+                    JSON.stringify(data[n].Data) == JSON.stringify(chart.series()[n].Data.rawData());
+            }
+            //Exit early if data has not changed
+            if (dataUnchanged)
+                return;
+
+            var defaultSettings = {
+                legend: false,
+                ordered: true,
+                columns: false
+            };
+            var settings = $.extend(defaultSettings, options);
+
+            // Hack to fix insight's lack of data reloading
+            $element.html('');
+
+            chart = new insight.Chart('', $element[0])
+                .width($element.width())
+                .height(400); //Math.min(barCount * 50 + 100, 400));
```

Comment:
The bars look a bit big when there are not many options, and squashed when there are. Can we scale this by the number of options/rounds?